server/models/User.js

```diff
@@ -110,6 +114,16 @@ class User {
         return this.blockList.includes(otherUser.username.toLowerCase());
     }
 
+    getFullDetails() {
+        let user = Object.assign({}, this.userData);
+
+        delete user.password;
```

Comment:
Is the password the only thing that needs to be removed? Isn't there also a list of current tokens?

Reply:
This is used in the admin page, which actually uses the session info, so this is ok